runtime/README.md

```diff
@@ -155,14 +152,6 @@ Table: `ECDSA384_SIGNATURE_VERIFY` input arguments
 | signature\_r | u8[48]   | R portion of signature to verify
 | signature\_s | u8[48]   | S portion of signature to verify
 
```

Comment:
Add "`ECDSA384_SIGNATURE_VERIFY` returns no output arguments." for consistency?

Reply:
Done